### DISCUSSION TITLE
How to change prompt colors?

### DISCUSSION THREAD

**Discussion by K2ouMais** [starship/starship#1819] [Q&A] [answered]
Hello everyone,

is there a way to change the prompt colors?

![Starship](https://user-images.githubusercontent.com/52510455/96853316-7ce61e00-145a-11eb-9cfb-1121c054896c.png)

I am seeing a difference between the colors at home and the colors at the work.

The only diference is at home I am using WSL2 and at work WSL1.

At home the directory colors and the php colors are different, but at the work they are the same. (screenshot)

I am using Cmder in Windows 10 with the same font and the same theme.

**Comment by mickimnet** [comment, marked as answer]
The `directory` module is using Cyan as a color and the `PHP` module is using 147 as color. They are not the same ;) If they may look same, that can also depend on the display and the setting of those displays.

In order to change the colors, you can create a `starship.toml` file and use the `$STARSHIP_CONFIG` environment variable, containing the full path including the file name of the configuration file.

Then place the following in the file:
```toml
[directory]
style = "cyan bold"

[php]
style = "147 bold"
```

"Cyan" and "147" are the color values. Change them to whatever you prefer. For more details go the [Advanced configuration -> Style Strings](https://starship.rs/advanced-config/#style-strings) section of the documentation.
